src/components/BookList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Book from './Book';
+
+export default function BookList() {
+  return (
+    <>
+      <Book />
+    </>
+  );
+}
```

Comment:
- [ ] You can consider either keeping the BookList file with the return statement to render the data or the Book.js file instead of adding the Book component in the BookList.